java/generate_jni_header_files.sh

```diff
@@ -40,7 +40,13 @@ generate_one io.ray.runtime.actor.NativeActorHandle
 generate_one io.ray.runtime.object.NativeObjectStore
 generate_one io.ray.runtime.task.NativeTaskExecutor
 generate_one io.ray.runtime.gcs.GlobalStateAccessor
+generate_one io.ray.runtime.metric.TagKey
+generate_one io.ray.runtime.metric.Metric
+generate_one io.ray.runtime.metric.Gauge
+generate_one io.ray.runtime.metric.Count
+generate_one io.ray.runtime.metric.Sum
+generate_one io.ray.runtime.metric.Histogram
```

Comment:
We can put all native methods in one util class, so we just need to maintain 1 JNI file.